content/_data/changelogs/weekly.yml

```
@@ -15605,6 +15605,110 @@
   # pull: 6545 (PR title: Deprecate `Functions#getServerName`)
   # pull: 6548 (PR title: Bump executable-war from 2.4 to 2.5)
 
+  - version: '2.348'
+    date: 2022-05-15
+    changes:
+      - type: rfe
+        category: developer
+        pull: 6538
+        authors:
+          - janfaracik
+        pr_title: Add color override classes
+        message: |-
+          Developer: New series of colours and new range of semantic colours added (see [Design Library](https://plugins.jenkins.io/design-library/) for more details).
+      - type: rfe
+        category: rfe
+        pull: 6571
+        authors:
+          - basil
+        pr_title: Use `URLClassLoader` by default
+        message: |-
+          Use native Java Platform functionality rather than Ant to load classes. The old behavior can be restored by setting <code>-Dhudson.ClassicPluginStrategy.useAntClassLoader=true</code>.
+      - type: rfe
+        category: rfe
+        pull: 6565
+        authors:
+          - dependabot[bot]
+        pr_title: Upgrade Spring Framework from 5.3.19 to 5.3.20
+        message: |-
+          Upgrade [Spring Framework](https://spring.io/projects/spring-framework) from 5.3.19 to 5.3.20. Spring Framework 5.3.20 includes [14 fixes and improvements](https://github.com/spring-projects/spring-framework/releases/tag/v5.3.20). In addition, this releases include fixes for 2 vulnerabilities: [CVE-2022-22970](https://tanzu.vmware.com/security/cve-2022-22970) and [CVE-2022-22971](https://tanzu.vmware.com/security/cve-2022-22971).
+      - type: rfe
+        category: rfe
+        pull: 6543
+        issue: 51820
+        authors:
+          - jglick
+          - timja
+        pr_title: "[JENKINS-51820] Removing Java Web Start support"
+        message: |-
+          Remove [Java Web Start](https://en.wikipedia.org/wiki/Java_Web_Start) support for launching inbound agents, along with the GUI mode, the platform-specific agent installers and the JAR signature.
```

Hyperlink needs to be moved into a references section
```suggestion
          Remove [Java Web Start](https://en.wikipedia.org/wiki/Java_Web_Start) support for launching inbound agents, along with the GUI mode, the platform-specific agent installers and the JAR signature.
```
```suggestion
          Remove [Java Web Start](https://en.wikipedia.org/wiki/Java_Web_Start) support for launching inbound agents, along with the GUI mode, the platform-specific agent installers and the JAR signature.
```